scripts/flatpak.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -xe
```

Comment:
I referenced this and the json manifest file from `wezterm`. Huge shoutout there, it was a big help!